### PR TITLE
HKYawdAT: Add integration test for disconnected IDP hints

### PR DESCRIPTION
**Pull request by bjgill**
See https://trello.com/c/HKYawdAT/133-add-integration-test-for-disconnected-idp-hints

Co-authored-by: Alex Newton <alex.newton@digital.cabinet-office.gov.uk>

### CODE REVIEW

**Comment by bjgill** [comment]
The new test added by this PR is somewhat flaky when run locally. However, it fails in the same way (and the a similar frequency) to the existing pause and resume tests. The P&R tests don't seem to be flaky when used in staging/joint/etc. for the smoke tests. 

Thus, my hypothesis is that this new test - despite being flaky when run locally - will be fine when run in staging/joint/etc.

Hence, I propose to merge this PR and then watch staging/joint closely for failures that would require us to back this out.

@phantas - pinging you in case you have a view

**Comment by bjgill** [comment]
OK. I've fixed the main cause of the flakiness. I think the rest was just caused by my machine running out of RAM. I think this is now ready for reviews. Once stub-idp has been updated, this PR can then be merged.

- [ ] Get changes to stub IDP released.

File: features/simple_sign_in.feature [hunk 1]
```diff
@@ -13,7 +13,23 @@ Feature: User simple flows - sign in
   Scenario: User cannot sign in using a disconnected IDP
     Given the user is at Test RP
     And they start a sign in journey
-    Then they cannot sign in with IDP "Stub Idp Demo Three"
+    Then they cannot sign in with IDP "Stub Idp Disconnected"
+
+  Scenario: User can see disconnected IDP hint
+    Given the user is at Test RP
+    And they start a registration journey with IDP "Stub Idp Disconnected"
+    And they submit user details:
```
Comment (galund) [inline features/simple_sign_in.feature]: I guess I'm surprised that we have to repeat this boiler-plate everywhere, but fine

Comment (bjgill) [reply features/simple_sign_in.feature]: I'm not thrilled either, but I couldn't find an easy way of putting this into a step.

**Pull request merged**